Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,25 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about diving?"
+    m 3wub "It's very exciting. Just imagine you are diving."
+    m 3wub "You can see a wonderful underwater world full of different marine life."
+    m 7eub "There you can see: schools of fish, coral reefs, and sea greens."
+    m 2wub "There may be sharks."
+    m 3esb "If you meet a shark, then here is one rule for you."
+    m 3esb "Do not make sudden movements."
+    m 2euc "Although meeting a shark can be dangerous for you."
```

Comment:
It should probably be noted that shark attacks are extremely rare and even then, they aren't much of a concern with safe and responsible diving. 

I'd bring up some notes about responsible diving such as not just diving anywhere and looking out for signs.